config.yaml

```diff
@@ -60,11 +60,22 @@ directories:
         # host, and that the module is configured to match the local path.
         # See code for details.
         archive:
-                rsyncd_module: plots
+                mode: remote  # remote or local
+                rsyncd_module: plots  # ignored in local mode
+
+                # Note for local mode:
```

Comment:
Maybe there should be a `local` mapping containing the local config rather than sharing `rsyncd_path`?  I guess local is still using rsync, not rsyncd.  Hmm...  I'm really not sure.  But, I'll leave this here to see how you feel.

In general, look at integrating the existing remote and new local comments with each other.

Reply:
I have implemented something like this ^ now.